httomo/wrappers_class.py

```diff
@@ -274,36 +281,19 @@ def __init__(
         super().__init__(module_name, function_name, method_name, comm)
 
         # if not changed bellow the generic wrapper will be executed
-        self.wrapper_method = super()._execute_generic
+        self.wrapper_method: Callable = super()._execute_generic
 
-        if module_name in ["misc", "prep", "recon"]:
+        if module_name in ["misc"]:
             from importlib import import_module
 
             self.module = getattr(
                 import_module("httomolib." + module_name), function_name
             )
             if function_name == "images":
                 self.wrapper_method = self._execute_images
-            if function_name == "normalize":
-                func = getattr(self.module, method_name)
-                sig_params = signature(func).parameters
-                if "darks" in sig_params and "flats" in sig_params:
-                    self.wrapper_method = super()._execute_normalize
-            if function_name == "algorithm":
-                self.wrapper_method = super()._execute_reconstruction
-            if function_name == "rotation":
-                self.wrapper_method = super()._execute_rotation
-
-        # httomolib can include GPU/CuPy methods as as well as the CPU ones. Here
-        # we check if the method can accept CuPy arrays by looking into docstrings.
 
-        # get the docstring of a method in order to check the I/O requirements
-        get_method_docs = inspect.getdoc(getattr(self.module, method_name))
-        # if the CuPy array mentioned in the docstring then we will enable
-        # the GPU run when it is possible
+        # httomolib methods run on the CPU only
         self.cupyrun = False
```

Comment:
`self.cupyrun` is `False` already from the `BaseWrapper`, we don't have to assign it to False in the derived classes.